src/backend/expungeservice/models/charge_types/traffic_non_violation.py

```diff
@@ -8,8 +8,8 @@
 class TrafficNonViolation(Charge):
```

Comment:
This may be a dumb question, but what is a TrafficNonViolation Charge?

Reply:
It is a traffic crime that is more severe than a violation.